src/mds/MDSDaemon.cc

```diff
@@ -340,6 +351,11 @@ void MDSDaemon::set_up_admin_socket()
                                      asok_hook,
                                      "show cache status");
   ceph_assert(r == 0);
+  r = admin_socket->register_command("quiesce subvolume "
```

Comment:
I'd suggest to have this command as `quiesce path` to abstract away from subvolumes. Whether a path is "quiesceable" will then be an internal decision. We may want to add arbitrary path quiesce at some point